substra/sdk/backends/local/compute/spawner.py

```diff
@@ -48,13 +49,14 @@ class ExecutionError(Exception):
 class DockerSpawner:
     """Wrapper around docker daemon to execute a command in a container."""
 
-    def __init__(self):
+    def __init__(self, local_worker_dir: pathlib.Path):
         self._docker = docker.from_env()
         self._user = os.getuid()
+        self._wdir = local_worker_dir
 
     def spawn(self, name, archive_path, command, volumes=None, envs=None):
         """Spawn a docker container (blocking)."""
-        with tempfile.TemporaryDirectory() as tmpdir:
+        with tempfile.TemporaryDirectory(prefix=str(self._wdir) + "/") as tmpdir:
```

Comment:
`TemporaryDirectory` has a `dir` arg. I think we could use it instead of prefix, this way there's no need to add a trailing slash to convert to the prefix into a path.

Reply:
Thank you ! updated it